dts/bindings/serial/uart-rs485-controller.yaml

```diff
@@ -0,0 +1,41 @@
+#Common fields for uart-rs485 controllers
+
+include: uart-controller.yaml
+
+properties:
+    rs485-enabled:
+      type: boolean
+      required: false
+      description: Enable rs485 mode
```

Comment:
Considering `UART_LINE_CTRL_RS485 = BIT(5),` and Kconfig option `SERIAL_SUPPORT_RS485`, this property does not seem necessary to me. Ideally, uart-rs485-controller.yaml should only describe bindings to a RS-485 transceiver. If Kconfig optoin `SERIAL_SUPPORT_RS485` is not enabled, attempt to set `UART_LINE_CTRL_RS485` should probably return an -ENOTSUP (API not enabled).

It would be nice to have optional bindings for the GPIOs if the controller does not have RS-485 and dedicated pins for DE and RE, see spi-controller.yaml as an example.